core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/console/CodeSystemsCommand.java

```diff
@@ -81,21 +81,77 @@ public void run(CommandLineStream out) {
 	}
 
 	private String getCodeSystemInfo(CodeSystem codeSystem) {
```

Comment:
Let's chat about the amount of manual formatting required in this method and what can we do about it.

Reply:
I remade the foramtting